lib/src/clojure_lsp/shared.clj

```diff
@@ -435,5 +435,8 @@
     {:start {:line (max 0 (dec (or row name-row))) :character (max 0 (dec (or col name-col)))}
      :end {:line (max 0 (dec (or end-row name-end-row))) :character (max 0 (dec (or end-col name-end-col)))}}))
 
+(def full-file-position
+  {:row 1 :col 1 :end-row 1000000 :end-col 1000000})
+
 (defn full-file-range []
```

Comment:
I'm okay just changing this to a `def`

Reply:
Ah, yes, I'll plan on doing that in a future PR.